datahub-web-react/src/app/ingest/source/builder/RecipeForm/constants.ts

```diff
@@ -78,6 +78,23 @@ import { POSTGRES } from '../../conf/postgres/postgres';
 import { POSTGRES_HOST_PORT, POSTGRES_DATABASE, POSTGRES_USERNAME, POSTGRES_PASSWORD } from './postgres';
 import { HIVE } from '../../conf/hive/hive';
 import { HIVE_HOST_PORT, HIVE_DATABASE, HIVE_USERNAME, HIVE_PASSWORD } from './hive';
+import {
+    LOOKML,
+    CONNECTION_TO_PLATFORM_MAP,
+    DEPLOY_KEY,
+    LOOKML_BASE_URL,
+    LOOKML_CLIENT_ID,
+    LOOKML_CLIENT_SECRET,
+    LOOKML_GITHUB_INFO_REPO,
+    PARSE_TABLE_NAMES_FROM_SQL,
+    PROJECT_NAME,
+} from './lookml';
+
+export enum RecipeSections {
+    Connection = 0,
+    Filter = 1,
```

Comment:
Is filter being used anywhere? What is it for?

Reply:
yup! most forms have the filter section - it's where you add allow/deny patters